packages/server/src/adapters/node-http/content-type/form-data/index.ts

```diff
@@ -34,7 +33,22 @@ async function parseMultipartFormData(
 
   const formData = new FormData();
   const parts: AsyncIterable<UploadHandlerPart & { done?: true }> =
-    streamMultipart(Readable.toWeb(request), boundary);
+    streamMultipart(
+      new ReadableStream(
```

Comment:
Isn't this also a node17+ class? In the original issue I'm definitely listing it as such

Reply:
> Isn't this also a node17+ class?

[`Readable.toWeb`](https://nodejs.org/api/stream.html#streamreadabletowebstreamreadable-options)/[`Readable.fromWeb`](https://nodejs.org/api/stream.html#streamwritablefromwebwritablestream-options) definitely are. However, [`ReadableStream`](https://nodejs.org/docs/latest-v16.x/api/webstreams.html#class-readablestream) has been available since `16.5.0`